packages/store/schematics/factories/ng-add/ng-add.factory.ts

```diff
@@ -32,28 +52,66 @@ function addNgxsPackageToPackageJson() {
 function runNpmPackageInstall() {
   return (_: Tree, context: SchematicContext) => {
     context.addTask(new NodePackageInstallTask());
-    context.logger.log('info', `🔍 Installing packages...`);
+    context.logger.info(`🔍 Installing packages...`);
   };
 }
 
-// function setSchematicsAsDefault(): Rule {
-//   return (host: Tree, context: SchematicContext) => {
-//     context.logger.info('Adding @ngxs/schematics to angular.json');
-//     const exec = require('child_process').exec;
-//     exec('ng config cli.defaultCollection @ngxs/schematics', () => {
-//       context.logger.log('info', `✅️ Setting NGXS Schematics as default`);
-//     });
-//     return host;
-//   };
-// }
-
 function finallyLog(): Rule {
   return (host: Tree, context: SchematicContext) => {
-    context.logger.log(
-      'info',
+    context.logger.info(
       `👏 Create your first ngxs store by using starter kit: ng g ngxs-sk --spec`
     );
 
     return host;
   };
 }
+
+function addDeclarationToNgModule(options: NgxsPackageSchema): Rule {
+  return (host: Tree, context: SchematicContext) => {
+    if (typeof options.name === 'undefined' || options.name === '') {
+      return host;
+    }
+
+    const project = getProject(host, options.name);
+
+    context.logger.info(JSON.stringify(project));
+
+    if (typeof project === 'undefined' || project === null) {
+      throw new SchematicsException(`Project "${options.name}" does not exist.`);
+    }
+
+    const modulePath = getAppModulePath(host, `${project.root}/src/main.ts`);
+
+    context.logger.info(`module path, ${modulePath}`);
+
+    if (typeof modulePath === 'undefined') {
+      throw new SchematicsException(
+        `Module path for project "${options.name}" does not exist.`
+      );
+    }
+
+    const importPath = '@ngxs/store';
+
+    const moduleImport =
+      'NgxsModule.forRoot([], {developmentMode: !environment.production,selectorOptions: {suppressErrors: false,injectContainerState: false}})';
```

Comment:
Slight format tweaks, so that prettier has less work to do 😉 
```suggestion
      'NgxsModule.forRoot([], { developmentMode: !environment.production, selectorOptions: { suppressErrors: false, injectContainerState: false } })';
```